src/AppBundle/Entity/Topic.php

```diff
@@ -185,7 +185,12 @@ public function getDescription()
      */
     public function getVotes()
     {
-        return $this->votes->toArray();
+        return array_values(array_filter(
```

Comment:
@smoench right, that would have been more elegant, thx - I will refactor this later